db/migrate/20220909120838_correct_referral_value.rb

```diff
@@ -0,0 +1,5 @@
+class CorrectReferralValue < ActiveRecord::Migration[7.0]
+  def change
+    LettingsLog.where(referral: 0).update_all(referral: 1)
```

Comment:
Is this reversible? 

I think it's generally best practice to only use migrations for schema changes, not for data changes - not sure it really matters hugely in this case but I think using rake tasks for data changes is a good idea.

Reply:
Good point! I don't know much about Rails so didn't realise this was the sort of thing you could do with a rake task. I've changed it to one since it's not much effort.